app/templates/skeleton/drupal-org.make

```diff
@@ -97,6 +97,11 @@ projects[views][version] = "3.13"
 projects[views_bulk_operations][subdir] = "contrib"
 projects[views_bulk_operations][version] = "3.3"
 
+; Themes
+projects[bootstrap][subdir] = "contrib"
+projects[bootstrap][version] = "3.x"
```

Comment:
Thanks, but this is still wrong. Version should be locked. since beta-2 was created long ago, you can specify the git commit Id.